internal/app/controllers/geometry.go

```diff
@@ -31,6 +31,18 @@ func (u GeometryController) Cover(c *gin.Context) {
 	var s2cells [][][]float64
 
 	for _, f := range fs {
+		if f.Geometry.IsMultiPolygon() {
+			for _, mp := range f.Geometry.MultiPolygon {
+				for _, p := range mp {
```

Comment:
It might be better to export this inner iteration to its own private function to avoid duplication

Reply:
refactored thanks